paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -34,12 +32,13 @@ fun detectEnvironment(): Environment {
   val userDir = System.getProperty("user.dir")
   val userHome = System.getProperty("user.home")
   val androidHome = System.getenv("ANDROID_HOME") ?: "$userHome/Library/Android/sdk"
-  val platformDir = Files.list(Paths.get("$androidHome/platforms"))
-      .filter { Files.isDirectory(it) }
+  val platformDir = File("$androidHome/platforms")
+      .listFiles()
+      .orEmpty()
+      .filter { it.isDirectory }
       .map { it.toString() }
       .sorted()
-      .reduce { _, next -> next }
-      .orElse(null)
+      .last()
```

Comment:
`sorted().last()` is basically `max()` but I don't think it's any more readable... 

Also, that's a very edge casy but, sorting platforms by string value doesn't seem reliable way of getting latest platform (I assume that's what is wanted here). If I had `android-9` installed (lol) it would use that as a latest platform... I'm wondering if we could use `sdkmanager` cli for a more deterministic approach but that would just slow things down. Alternatively we can maintain a map of known platform names and specify the order there but that would require maintaining it every year or so. As I said, that's an edge case and as long as there are no 9- platforms installed should work fine.